templates/tfengine/components/cicd/main.tf

```diff
@@ -51,7 +51,6 @@ data "google_project" "devops" {
 }
 
 locals {
-  cloudbuild_sa = "serviceAccount:${data.google_project.devops.number}@cloudbuild.gserviceaccount.com"
```

Comment:
We should still support for the case where users are using the default service account. Also we should make it a non-breaking changes for those users (who use the default service account).

Reply:
On a second thought, maybe we can enforce the behaviour of always using a custom SA as it's generally a better practice. Let's chat about it a bit more tomorrow.

Reply:
SA and logs_bucket are added to required fields, resolving.